### DISCUSSION TITLE
How to get 𝐘𝐨𝐥𝐨 and 𝐏𝐮𝐥𝐥 𝐒𝐡𝐚𝐫𝐤 EASIEST 😉🖤(Until September only!)

### DISCUSSION THREAD

**Discussion by Guedesou** [Schweinepriester/github-profile-achievements#50] [Ideas]
Hi, people!🌎 To help us, I decided to write these steps and believe me: **even if you are a 'padawan' this path may be the EASIEST and FASTEST way to do it:**
----
0) Please follow me on GitHub? I follow you all back and it will be a pleasure to have you in my network.

***[GitHub](https://github.com/Guedesou) | [LinkedIn](https://www.linkedin.com/in/guedesou/)***
----
1) There is a course called _Introduction to GitHub_ which is hosted by GitHub Lab. GitHub Lab will be maintained by the owners until September 1 (one) as widely announced on the page's HEAD _![labshutdown](https://user-images.githubusercontent.com/56264781/175278105-72cba643-10a5-4802-b794-367e797f28d8.PNG)_

[Link HERE](https://lab.github.com/githubtraining/introduction-to-github?overlay=register-box-overlay)
----
2) _Make the entire course_ (in addition to learning about basic tools of GitHub, which is really cool for those who don't know or need to update themselves) and you will see these two in your profile! EASIEST! I told you so! 🤩🐙🧙🏼‍♂️
----

3) Enjoy and go for others achievements! You can make it and I approve and support your effort!
Please, don´t forget to [follow me here](https://github.com/Guedesou) and on [LinkedIn](https://www.linkedin.com/in/guedesou/), folk! 
I need some support too! ☜(ﾟヮﾟ☜)
----
![perfil](https://user-images.githubusercontent.com/56264781/175279541-58915687-286a-46ef-9446-5075b5ab9e79.PNG)

**Comment by smir45** [comment]
For getting YOYO its simple
1. Create one repository and open random pull request on it
2. merge it on your own
3. You got and YO-LO

**Reply by Guedesou** [reply]
### Yoyo? I didnt know her! 🤣
The Title is: How to get 𝐘𝐨𝐥𝐨 and 𝐏𝐮𝐥𝐥 𝐒𝐡𝐚𝐫𝐤 more easy.
Between seeding education, courses and knowledge or just a hack, I prefer people to know what they are doing, how to use the tools and GitHub itself.
Have a good week!
![EfvVgguWAAEvRv4](https://user-images.githubusercontent.com/56264781/175394983-df2d3269-589c-4bb1-955f-515842052bf7.jpg)

**Comment by serAnkii** [comment]
Just simply create a pull-request on your own repo instead of directly commiting the changes to the master branch , do it two times  and you will get the badge. 
hope it will help you.
